src/common/components/layouts/pages/inspect/errorMessages/english/validationErrorMessages_en.json

```diff
@@ -348,6 +348,10 @@
       "11": {
         "SUMMARY": "Document is not tagged",
         "DESCRIPTION": "This is a critical accessibility error. If the document has content that is not tagged, reader software for people with visual impairments may have difficulty reading the document out in the right order. You can probably solve this problem by re-exporting your document from your office application (Word or LibreOffice) as tagged PDF. You do this by following the menu 'File -> Save as -> (select PDF) -> Options' and ticking the options 'Tagged PDF' and 'Save as ISO 19005-1 (PDF/A)'."
+      },
+      "12": {
+        "SUMMARY": "A Structure element dictionary does not contain the P (parent) entry",
+        "DESCRIPTION": "Structure element dictionary shall contain the P (parent) entry"
```

Comment:
Structure element is missing the reference to the parent, so it can not be reached from the structure root.